Fuchas/Kernel/boot.lua

```diff
@@ -106,7 +106,7 @@ function gy() -- temporary cursor Y accessor
 end
 
 local lastFore = 0
-local function write(msg, fore)
+function _G.write(msg, fore)
```

Comment:
there is a reason for it being a local function, it is so that it can be used even after `shell` replaces the `write(...)` function, so that in case of error in shell the BSOD can still trigger correctly.

Reply:
oh ok that makes sense